services/discovery/discovery-utils.js

```diff
@@ -14,25 +14,29 @@
  * limitations under the License.
  **/
 const pkg = require('../../package.json'),
-  DiscoveryV1 = require('watson-developer-cloud/discovery/v1');
+  DiscoveryV1 = require('ibm-watson/discovery/v1'),
+  { IamAuthenticator } = require('ibm-watson/auth');
+
 
 function DiscoveryUtils() {}
 DiscoveryUtils.prototype = {
 
   buildService: function(username, password, apikey, endpoint) {
+    let authSettings = {};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4846228)